cmd/wallet-js-sdk/test/specs/common.js

```diff
@@ -52,13 +52,15 @@ export async function loadFrameworks({
   mediaTypeProfiles = ["didcomm/aip2;env=rfc587", "didcomm/aip2;env=rfc19"],
   keyType = "ed25519",
   keyAgreementType = "p256kw",
+  onlyWallet = false,
```

Comment:
what is the use of this flag?

Reply:
It used to load agent-vcwallet-js-worker.wasm instead agent-js-worker.wasm